custom_dc/build_image.md

```diff
@@ -41,82 +41,79 @@ If you want to pick up the latest prebuilt version, do the following:
 - You have a Github account and project.
-- You have created a fork off the base Data Commons `website` repo (https://github.com/datacommonsorg/website){: target="_blank"} and a remote that points to it, and that you will push to that fork. 
-
+- You have created a fork off the base Data Commons `website` repo [https://github.com/datacommonsorg/website](https://github.com/datacommonsorg/website){: target="_blank"}, which you will use to develop with. 
 
 ### Sync a local workspace to the stable release
 
-If you are using a version control system other than Github, you can download a ZIP or TAR file from [https://github.com/datacommonsorg/website/releases/tag/customdc_stable](https://github.com/datacommonsorg/website/releases/tag/customdc_stable){: target="_blank"}. 
-
-In Github, use the following procedure.
+#### Clone the stable branch only
 
-1. If you want to reuse the root directory you previously created and cloned, skip to step 3. 
-If you want to create a new source directory and start from scratch, clone the repo up to the stable release tag:
+Use this procedure if you are not using Github, or if you are using Github and want to create a new source directory and start from scratch. If you are using Github and want to use the same directory and files you previously cloned, skip to the next section.
 
+1. Run the following command:
    <pre>
-   git clone https://github.com/datacommonsorg/website --branch customdc_stable --single-branch  [<var>DIRECTORY</var>]
+   git clone https://github.com/datacommonsorg/website --b customdc_stable --single-branch  [<var>DIRECTORY</var>]
    </pre>
-1. Change to the root directory:
-
+   This creates a new local branch called `customdc_stable` set to track the Data Commons repo branch. 
+1. To verify, run:
    <pre>
    cd website | cd <var>DIRECTORY</var>
+   git branch -vv
    </pre>
+   You should see output like the following:
 
-1. Create a new branch synced to the stable release:
+   ```
+   * customdc_stable 83732891 [origin/customdc_stable] 2024-11-06 Custom DC stable release (#4710)
+   ```
+   Rather than developing on this default branch, we recommend that you create another branch, as described in the next step.
+
+#### Create a remote and sync a local branch to the stable branch
+This procedure assumes that you are using Github.
 
+1. From the `website` or other directory where you have cloned the code, create a remote that points to the Data Commons repo, and only uses the stable branch:
    <pre>
-   git checkout -b <var>BRANCH_NAME</var> customdc_stable
+   git remote add -t customdc_stable <var>REMOTE_NAME</var> https://github.com/datacommonsorg/website
    </pre>
-
-1. To verify that your local repo is at the same version of the code, run the following command:
-
+1. Create a new branch synced to the stable branch:
+   <pre>
+   git checkout -b <var>BRANCH_NAME</var> origin/customdc_stable
```

Comment:
Since the remote name is hard-coded here, how about we recommend something for it both here (line 86) and above (line 82)? I'd recommend calling it `upstream` for consistency with our other docs and tools.

Reply:
Yeah, that's a good point, I do what you suggest, which is that origin is my fork and upstream is the original. Let me rewrite this accordingly -- including mentioning that we assume that they keep the default fork name of "origin", in which case this part should actually their remote name, right?

Reply:
Actually, no I just realized that this is the opposite of what you get by default when you run git clone.

Reply:
So in reality, they don't even need to this step at all!